pkg/config/config.go

```diff
@@ -732,8 +716,19 @@ func (c *config) Load(path string, fsys fs.FS) error {
 		} else if undecoded := metadata.Undecoded(); len(undecoded) > 0 {
 			fmt.Fprintf(os.Stderr, "Unknown config fields: %+v\n", undecoded)
 		}
+		if base.ProjectId == c.baseConfig.ProjectId {
+			fmt.Fprintf(os.Stderr, "WARN: project_id is missing for remote %s this config won't apply to any branch\n", name)
```

Comment:
```suggestion
			fmt.Fprintf(os.Stderr, "WARN: project_id is missing for [remotes.%s]\n", name)
```

There's an edge case where user may declare `project_id = "<project_ref>` in the base config, which means the id is inherited over. Let's just not mention that it won't apply to any branch.